public/docs/content/troubleshooting/fix-cdk-version.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Custom CDK version
+tags: cdk, constructs, versioning, resolutions
+packages: nx-monorepo
+---
+
+# I want to use a fixed CDK/Constructs package version
+
+```ts
+const cdkVersion = ...;
+const constructsVersion = ...;
+
+monorepo.tryFindObjectFile("package.json")?.addOverride("resolutions", {
```

Comment:
Why wouldn't pinning the version directly in the deps for the project work?

Reply:
based on my experience (I may be wrong), adding the `aws-cdk-lib@<version>` as dependency will use the latest version